test/topology/test_topology.py

```diff
@@ -61,3 +66,69 @@ async def test_decommission_node_add_column(manager, random_tables):
     await manager.decommission_node(servers[1])             # Decommission [1]
     await table.add_column()
     await random_tables.verify_schema()
+
+
+@pytest.mark.asyncio
+async def test_remove_node_with_concurrent_ddl(manager, random_tables):
+    stopped = False
+    ddl_failed = False
```

Comment:
Another way could be to use an `asyncio.Queue` to signal an issue.

Reply:
just a bool is simpler, imho